.github/workflows/nestbuildmatrix.yml

```diff
@@ -621,19 +621,18 @@ jobs:
       matrix:
         os: [macos-latest]
         cpp_compiler: ["clang"]
-        NEST_BUILD_TYPE: ["FULL_MACOS"]
+
+        # available use flags (all default to "OFF"):
+        # openmp, mpi, python, gsl, ltdl, boost, sionlib, libneurosim, optimize, warning, userdoc, music
+        use:
+          - "openmp, mpi, python, gsl, ltdl, boost, optimize, warning"
```

Comment:
```suggestion
          - "openmp, mpi, python, gsl, ltdl, boost, hdf5, optimize, warning"
```

Reply:
@nicolossus That should be fixed now.

Reply:
@heplesser LGTM. You may resolve the conversation.